packages/assert/package.json

```diff
@@ -9,7 +9,7 @@
         "build": "tsc && copyfiles -u 3 './lib/src/monorepo_scripts/**/*' ./scripts",
         "clean": "shx rm -rf lib test_temp scripts",
         "lint": "tslint --project . 'src/**/*.ts' 'test/**/*.ts'",
-        "run_mocha": "mocha lib/test/**/*_test.js",
+        "run_mocha": "mocha lib/test/**/*_test.js --exit",
```

Comment:
Do you know why that happens? Ideally - we would not need that flag if we properly stop the provider in the global after all hook but I might be too lazy to do that ;)

Reply:
Oh really? So we can just stop it in a global after hook?